pandas/tests/frame/indexing/test_where.py

```diff
@@ -1017,3 +1020,15 @@ def test_where_producing_ea_cond_for_np_dtype():
         {"a": Series([pd.NA, pd.NA, 2], dtype="Int64"), "b": [np.nan, 2, 3]}
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "replacement", [0.001, True, "snake", DATETIME_JAN_1_1900_OPTIONAL_TZ]
```

Comment:
` DATETIME_JAN_1_1900_OPTIONAL_TZ` is gonna be a `LazyStrategy` object here, which I guess isn't quite what you wanted, I think you'll need `@given` to use hypothesis tests
Not sure we need that though, just some date should be fine if you want to test using a date as a replacement

Also, the original issue used `None`, shall we include that?

Reply:
You're right, I didn't want a LazyStrategy object! I'll change that to a date, and also include None to align better with the original issue. I was also wondering if there was already a quick way to test all scalar types rather than having to use a long parametrized list. Or would that be overkill?

Reply:
yeah I think just keeping the original snippet from the issue as a test should be fine